readthedocs-theme/templates/readthedocs/pricing.html

```diff
@@ -0,0 +1,504 @@
+          </div>
+        </div>
+        <!-- end card -->
+        
+        <!-- Card -->
+        <div class="ui inverted card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon yellow big">Pro</div>
+
+            <div class="ui header huge">$250</div>
+            <div class="meta">
+              <span class="category">per month</span>
+            </div>
+
+            <button class="ui button fluid large yellow">Subscribe</button>
+
+            <div class="description">
+              
+              <div class="ui list inverted relaxed divided">
+                <div class="item">
+                  <div class="content">
+                    <div class="header"><i>All the features from the Advanced plan</i></div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">1 day support response time</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">6 Concurrent builds</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Search term analytics</div>
+                    <div class="description">90 days</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Pageview analytics</div>
+                    <div class="description">90 days</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Auth auditing</div>
+                    <div class="description">90 days</div>
+                  </div>
+                </div>
+                <div class="item">
+                  <div class="content">
+                    <div class="header">Single sign-on (SSO)</div>
+                    <div class="description">with Google</div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        </div>
+        <!-- end card -->
+
+      </div>
+
+      <div class="ui segment basic vertical padded">
+        <div class="ui horizontal divider header massive">
+          Base Plans
+        </div>
+      </div>
+      
+      <div class="ui three stackable cards">
+        
+        <!-- Card -->
+        <div class="ui card">
+
+          <div class="content">
+            
+            <div class="ui label ribbon blue big">Base</div>
+
+            <div class="ui header huge">Open Source</div>
+            <div class="meta">
+              <span class="category">forever free</span>
+            </div>
+
+            <button class="ui button fluid large blue">Signup</button>
```

Comment:
I left "Sign up" here because this one different from other plans in that it's not a subscription. But I think it's also ok to not highlight that and use the same call to action for all plans.